docs/how_tos/manual_testing.rst

```diff
@@ -8,9 +8,21 @@ Setting up for testing
 
 #. Make or refresh a copy of this repo where it can be seen from inside devstack: ``rsync -sax -delete ./ ../src/event-bus-kafka/`` (and rerun rsync after any edits as needed)
 #. In devstack, start Kafka and the control webapp: ``make dev.up.kafka-control-center`` and watch ``make dev.logs.kafka-control-center`` until server is up and happy (may take a few minutes; watch for ``INFO Kafka startTimeMs``)
+#. Sometimes kafka fails to start, run ``make dev.up.kafka-control-center`` again.
 #. Load the control center UI: http://localhost:9021/clusters and wait for the cluster to become healthy
-#. Create the topic you want to use for testing. For the examples used in the management commands, you'll want to use ``dev-user-login`` with default settings.
-#. In edx-platform's ``cms/envs/common.py``, add ``'edx_event_bus_kafka'`` to the ``INSTALLED_APPS`` list
+#. You can either create the topic you want to use for testing manually or perform the action in LMS or studio which will publish the event, creating the topic automatically. For the examples used in the management commands, you'll want to use ``dev-user-login`` with default settings.
+#. Update below settings in ``cms/envs/common.py`` or ``cms/envs/devstack.py``, make sure that they are defined only once.
+
+    .. code-block:: python
+
+            EVENT_BUS_PRODUCER = 'edx_event_bus_kafka.create_producer'
+            EVENT_BUS_CONSUMER = 'edx_event_bus_kafka.KafkaEventConsumer'
+            EVENT_BUS_KAFKA_SCHEMA_REGISTRY_URL = 'http://edx.devstack.schema-registry:8081'
+            EVENT_BUS_KAFKA_BOOTSTRAP_SERVERS = 'edx.devstack.kafka:29092'
+            EVENT_BUS_TOPIC_PREFIX = 'dev'
+            EVENT_BUS_KAFKA_CONSUMERS_ENABLED = True
```

Comment:
This setting is True by default by this point, so I think we can leave it out.

Reply:
Done.